src/geophires_x/Economics.py

```diff
@@ -1766,40 +1766,44 @@ def Calculate(self, model: Model) -> None:
+                # use SIMPLE approach
                 if hasattr(model.wellbores, 'Configuration'):
                     if model.wellbores.Configuration.value == Configuration.ULOOP:
-                        if hasattr(model.reserv,
-                                   'InputDepth'):  # must be using simple cylindrical model, which has an Input and Output Depth
+                        if hasattr(model.reserv, 'InputDepth'):
+                            # must be using simple cylindrical model, which has an Input and Output Depth
                             self.C1well = ((self.Vertical_drilling_cost_per_m.value *
-                                            (model.reserv.InputDepth.value * 1000.0)) +
+                                            (model.reserv.InputDepth.quantity().to('m').magnitude)) +
                                            (self.Vertical_drilling_cost_per_m.value * (
-                                                   model.reserv.OutputDepth.value * 1000.0)) +
+                                                   model.reserv.OutputDepth.quantity().to('m').magnitude)) +
                                            (
                                                    self.Nonvertical_drilling_cost_per_m.value * model.wellbores.Nonvertical_length.value)) * 1E-6
                         else:
                             if hasattr(model.wellbores, 'Nonvertical_length'):
                                 self.C1well = ((2 * self.Vertical_drilling_cost_per_m.value *
-                                                (model.reserv.depth.value * 1000.0)) +
+                                                (model.reserv.depth.quantity().to('m').magnitude)) +
                                                (
                                                        self.Nonvertical_drilling_cost_per_m.value * model.wellbores.Nonvertical_length.value)) * 1E-6
                             else:
                                 self.C1well = (2 * self.Vertical_drilling_cost_per_m.value * (
-                                        model.reserv.depth.value * 1000.0)) * 1E-6
-                    else:  # Coaxial
-                        self.C1well = ((self.Vertical_drilling_cost_per_m.value * (model.reserv.depth.value * 1000.0)) +
+                                        model.reserv.depth.value.quantity().to('m').magnitude)) * 1E-6
+                    else:
+                        # Coaxial
+                        self.C1well = ((self.Vertical_drilling_cost_per_m.value * (model.reserv.depth.quantity().to('m').magnitude)) +
                                        (
                                                self.Nonvertical_drilling_cost_per_m.value * model.wellbores.Nonvertical_length.value)) * 1E-6
 
+                else:
+                    self.C1well = self.Vertical_drilling_cost_per_m.value * model.reserv.depth.quantity().to(
+                        'm').magnitude * 1E-6
```

Comment:
[Implement missing simple well cost correlation for non-AGS wells >7km depth](https://github.com/NREL/GEOPHIRES-X/pull/126/commits/a463d1a7f9943a5e59fe49507bcd2796264f4a77)